var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -220,3 +220,20 @@ def check_install(self):
             source_file, [include_flags], link_flags.split()
         )
         compare_output_file(output, blessed_file)
+
+    @property
+    def blas_libs(self):
+        shared = '+shared' in self.spec
+        prefix = self.prefix
+        search_paths = [[prefix.lib, False], [prefix.lib64, False],
```

Comment:
Should this strategy actually be the default implementation of libs? Adding this to every package kind of defeats the purpose of default

@alalazo ping.